apps/meteor/client/omnichannel/reports/sections/TagsSection.tsx

```diff
@@ -6,17 +6,15 @@ import { ellipsis } from '../utils/ellipsis';
 
 export const TagsSection = () => {
 	const { data, ...config } = useTagsSection();
-	const { ref, contentBoxSize: { inlineSize: cardWidth = 200 } = {} } = useResizeObserver<HTMLElement>();
-	const width = cardWidth * 0.9;
+	const { ref } = useResizeObserver<HTMLElement>();
 
 	return (
 		<ReportCard ref={ref} {...config}>
 			<BarChart
 				data={data}
 				direction='horizontal'
 				height={360}
-				maxWidth={width}
-				margins={{ left: 40, top: 40, right: 20 }}
+				margins={{ left: 90, top: 30, right: 0 }}
```

Comment:
<img width="1345" height="503" alt="Image" src="https://github.com/user-attachments/assets/38492269-61cf-4229-b06e-cff505ccca8a" />

I noticed a labels being clipped still, depending on the combination of characters. For both departments and tags.

I believe at the time these charts were implemented, `left: 90` was defined with the maximum of 10+3 characters for each label in mind. What went under the radar is that Nivo seems to need `5px` for spacing.

I did a few tests and left margin of `92px` seems to be the number, since Nivo assigns `87px` for the text, leaving another `5px` free to be used for spacing. Do few tests and check if it covers all scenarios.

❗ BOT comment is relevant, you need that `8px` right margin otherwise the last number from that axis will be clipped.